sdk/tests/deqp/framework/opengl/gluShaderUtil.js

```diff
@@ -44,6 +44,11 @@ gluShaderUtil.GLSLVersion = {
 gluShaderUtil.getGLSLVersion = function(gl) {
     var glslversion = gl.getParameter(gl.SHADING_LANGUAGE_VERSION);
 
+    // TODO: Versions are not yet well implemented... Firefox returns GLSL ES 1.0 in some cases,
```

Comment:
Currently both mozilla and chrome report wrong GLSL versions (Mozilla reports GLSL ES 1.0, chrome reports GLSL ES 2.0). We ignore this error when testing internally. The error is not ignored in the released version of the test. Try running sdk/tests/deqp/functional/gles3/uniformbuffers.html. This TODO reminds us that we should turn the version check back on before creating a pull request. I can file bugs against the browsers, though.